Common/Settings.ini

```diff
@@ -280,6 +280,9 @@ WhiteShaderFix = 1
 ; Fixes spawning placements for objects in Woodside Apartments Room 205. (0|1)
 WoodsideRoom205Fix = 1
 
+; Fixes "Game Save" button audio. (0|1)
```

Comment:
Please place this fix below `RowboatAnimationFix` to keep in alphabetical order. Let's make its description a little more detailed:

```ini
; Restores the "game save" sound effect when selecting to save your game through the pause menu.
```